Frontend/ui-customer/src/Flow.purs

```diff
@@ -627,6 +628,11 @@ homeScreenFlow = do
                                       _ <- pure $ clearWaitingTimer state.props.waitingTimeTimerId
                                       homeScreenFlow
             "TRIP_FINISHED"       -> do -- TRIP FINISHED
+                                      (GetProfileRes response) <- Remote.getProfileBT ""
```

Comment:
@ashkriti Why getProfile api needs to be called after every trip completion, Can you add check on api call ?